### PR TITLE
DDF-2824 Remove corporate authors

### PR DESCRIPTION
**Pull request by brendan-hofmann**
#### What does this PR do?
Cleans up author tags referring to a corporation. Individual authors are retained.
#### Who is reviewing it? 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
#### Select relevant component teams: 
https://github.com/orgs/codice/teams
#### Ask 2 committers to review/merge the PR and tag them here. If you don't know who to ask, you can request reviews in https://groups.google.com/forum/#!forum/ddf-developers .
(please choose ONLY two committers from below, delete the rest)
@michaelmenousek
@shaundmorris
#### What are the relevant tickets?
[DDF-2824](https://codice.atlassian.net/browse/DDF-2824)
#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by brendan-hofmann** [comment]
build now

File: catalog/schematron/catalog-schematron-plugin/src/main/java/ddf/services/schematron/SchematronValidationException.java [hunk 1]
```diff
@@ -16,10 +16,7 @@
 import ddf.catalog.validation.impl.ValidationExceptionImpl;
 import java.util.List;
 
-/**
- * @author Shaun Morris, Lockheed Martin
- * @author ddf.isgs@lmco.com
- */
+/** @author Shaun Morris */
```
Comment (shaundmorris) [inline catalog/schematron/catalog-schematron-plugin/src/main/java/ddf/services/schematron/SchematronValidationException.java]: Nice I authored something!

File: platform/mime/core/platform-mime-core-api/src/main/java/ddf/mime/MimeTypeResolver.java [hunk 2]
```diff
@@ -17,7 +17,7 @@
  * Interface defining an OSGi service that can map a list of file extensions to their corresponding
  * mime types, and vice versa.
  *
- * @author Hugh Rodgers, Lockheed Martin
+ * @author Hugh Rodgers
  * @author ddf.isgs@lmco.com
```
Comment (shaundmorris) [inline platform/mime/core/platform-mime-core-api/src/main/java/ddf/mime/MimeTypeResolver.java]: Did you miss this `ddf.isgs@lmco.com`? Seems like you were removing them all except this one and the one in `TikaMimeTypeResolver.java`

**Comment by brendan-hofmann** [comment]
build now

**Pull request merged**